subiquity/models/filesystem.py

```diff
@@ -704,12 +707,21 @@ class Dasd:
     preserve: bool = False
 
 
+@fsobj("nvme_controller")
+class NVMeController:
+    transport: str
```

Comment:
This is a pretty finite list, correct?  An Enum would be nice.

Reply:
I guess we should consider what happens if there is a NVMe-oF devices of some other kind on the system.

Reply:
Yes, I'm not really sure what the other values could be. "rdma"  would probably be one of them but I haven't tested it.

Curtin's storage config will not complain if it encounters a value that it does not know about (it only knows about "tcp" really, not even "pcie"). Instead of raising a `ValueError` if the value for transport is unknown, I'd rather atempt the install. wdyt?

Reply:
I think I agree with your plan here